jt-classbreaks/src/main/java/it/geosolutions/jaiext/classbreaks/NaturalBreaksHistogramOpImage.java

```diff
@@ -166,5 +169,25 @@ protected void postCalculate(Classification c, int band) {
         }
         breaks[0] = buckets.get(0).getMin();
         hc.setBreaks(band, breaks);
+        if(percentages.booleanValue())
```

Comment:
jai-ext does not have auto-formatting, but could you still try to be consistent with gt/gs?

Reply:
fixed